systers_portal/templates/account/password_change.html

```diff
@@ -0,0 +1,21 @@
+{% extends "account/base.html" %}
+{% load url from future %}
+{% load i18n %}
```

Comment:
We don't do translations for Systers Portal. So you can remove the trans tags altogether.